docs/index.md

```diff
@@ -186,6 +186,33 @@ You can include Lambda Powertools Lambda Layer using [AWS Lambda Console](https:
         }
         ```
 
+    === "Pulumi"
+
+        ```typescript hl_lines="11"
+        import * as pulumi from "@pulumi/pulumi";
+        import * as aws from "@pulumi/aws";
+
+        const role = new aws.iam.Role("role", {
+            assumeRolePolicy: aws.iam.assumeRolePolicyForPrincipal(aws.iam.Principals.LambdaPrincipal),
+            managedPolicyArns: [aws.iam.ManagedPolicies.AWSLambdaBasicExecutionRole]
+        });
+
+        const lambdaFunction = new aws.lambda.Function("function", {
+            layers: [
+                pulumi.interpolate`arn:aws:lambda:${aws.getRegionOutput().name}:094274105915:layer:AWSLambdaPowertoolsTypeScript:3`
+            ],
+            code: new pulumi.asset.AssetArchive({
+                ".": new pulumi.asset.FileArchive("./app")
+            }),
+            tracingConfig: {
+                mode: "Active"
+            },
+            runtime: aws.lambda.Runtime.NodeJS16dX,
+            handler: "index.handler",
+            role: role.arn
```

Comment:
```suggestion
            role: role.arn,
            architectures: ["x86_64"]
```

Reply:
This is the default value for the architectures input, so you don't _have_ to have it there. Is it a best practice to include it?

Reply:
Yes indeed `x86_64` is the default value if not informed. it was just a suggestion to keep this option more explicit.

Reply:
Sorry, I meant that I think we set it by default behind the scenes if it's not set (presumably you do as well)